docs/template_plugin/tests/functional/subgraph_reference/preprocess.cpp

```diff
@@ -972,6 +972,25 @@ static RefPreprocessParams reverse_dyn_shape() {
     return res;
 }
 
+static RefPreprocessParams reverse_dyn_channels() {
+    RefPreprocessParams res("reverse_dyn_channels");
+    res.function = []() {
+        auto f = create_simple_function(element::u8, PartialShape{Dimension::dynamic(),
+                                                                  2,
+                                                                  Dimension::dynamic(),
+                                                                  Dimension::dynamic()});
+        auto p = PrePostProcessor(f);
+        p.input().tensor().set_layout("NCHW");
+        p.input().preprocess().reverse_channels();
+        p.build();
```

Comment:
why network layout is not set here? I supposed if tensor layout is set, then model layout must be set anyway

Reply:
No, here user just specifies that their input layout is NCHW. After this they can apply mean/scale/reverse_channels/whatever.
If 'network' layout is not specified, it is assumed as 'fully dynamic [...]' and no implicit conversion is needed